cli.js

```diff
@@ -30,7 +32,13 @@ if (!username) {
 
 	const createRow = (prefix, key) => {
 		if (user[key]) {
```

Comment:
I would recommend moving `user[key]` to a variable, since you are using it in many places.